secretkeylabs/xverse-web-extension
# refactor: updated ordinals tx functions

# 🔘 PR Type
What kind of change does this PR introduce?




# 📜 Background
Moved ordinal transactions logic to xverse-core

Issue Link: https://linear.app/xverseapp/issue/ENG-2551/updated-functions-for-ordinal-transactions
Context Link (if applicable): https://linear.app/xverseapp/issue/ENG-2528/move-ordinal-transactions-logic-to-xverse-core

# 🔄 Changes
Updated ordinals transactions functions from xverse-core
This PR depends on xverse-core PR https://github.com/secretkeylabs/xverse-core/pull/218

Impact:
- Send Ordinal, Restore Ordinal

Testing:
I have tested send Ordinal and recover Ordinal flow on iOS simulator. 

Testing Instruction: 

- Make sure Send Ordinal from collectibles is working fine including the fee calculation edit
- Make sure Recover Ordinals flow from settings is working fine including the fee calculation and fee edit

# 🖼 Screenshot / 📹 Video

https://github.com/secretkeylabs/xverse-web-extension/assets/29428363/af9dff1f-a479-471c-8950-4420df3b5df0



# ✅ Review checklist
Please ensure the following are true before merging:

- [x] Code Style is consistent with the project guidelines.
- [x] No unnecessary or debugging code has been added.
- [x] Security considerations have been taken into account.
- [x] The change has been manually tested and works as expected.
- [x] Breaking changes and their impacts have been considered and documented.
- [x] Code does not introduce new technical debt or issues.
